tests/importer/model_flattening.cpp

```diff
@@ -843,16 +843,21 @@ TEST(ModelFlattening, resolveFlattenCircularImportsComponents)
 {
     const std::string resolveError =
         "Cyclic dependencies were found when attempting to resolve a component in the model 'importExample2b'. The dependency loop is:\n"
-        " - component 'sideB' is imported from 'shared' in 'circularImport1.cellml';\n"
-        " - component 'shared' is imported from 'circular2' in 'circularImport2.cellml';\n"
-        " - component 'circular2' is imported from 'shared' in 'circularImport1.cellml'; and\n"
-        " - component 'shared' is imported from 'circular2' in 'circularImport2.cellml'.";
+        " - component 'sideB' specifies an import from ':this:' to '"
+        + resourcePath("modelflattening/") + "circularImport1.cellml';\n"
+                                             " - component 'shared' specifies an import from '"
+        + resourcePath("modelflattening/") + "circularImport1.cellml' to '" + resourcePath("modelflattening/") + "circularImport2.cellml'; and\n"
+                                                                                                                 " - component 'circular2' specifies an import from '"
+        + resourcePath("modelflattening/") + "circularImport2.cellml' to '" + resourcePath("modelflattening/") + "circularImport1.cellml'.";
+
     const std::string flattenError =
         "Cyclic dependencies were found when attempting to flatten a component in the model 'importExample2b'. The dependency loop is:\n"
-        " - component 'sideB' is imported from 'shared' in 'circularImport1.cellml';\n"
-        " - component 'shared' is imported from 'circular2' in 'circularImport2.cellml';\n"
-        " - component 'circular2' is imported from 'shared' in 'circularImport1.cellml'; and\n"
-        " - component 'shared' is imported from 'circular2' in 'circularImport2.cellml'.";
+        " - component 'sideB' specifies an import from ':this:' to '"
+        + resourcePath("modelflattening/") + "circularImport1.cellml';\n"
+                                             " - component 'shared' specifies an import from '"
+        + resourcePath("modelflattening/") + "circularImport1.cellml' to '" + resourcePath("modelflattening/") + "circularImport2.cellml'; and\n"
+                                                                                                                 " - component 'circular2' specifies an import from '"
+        + resourcePath("modelflattening/") + "circularImport2.cellml' to '" + resourcePath("modelflattening/") + "circularImport1.cellml'.";
```

Comment:
Replace with:
```c++
const std::string resolveError =
    std::string("Cyclic dependencies were found when attempting to resolve a component in the model 'importExample2b'. The dependency loop is:\n")
    + " - component 'sideB' specifies an import from ':this:' to '" + resourcePath("modelflattening/") + "circularImport1.cellml';\n"
    + " - component 'shared' specifies an import from '" + resourcePath("modelflattening/") + "circularImport1.cellml' to '" + resourcePath("modelflattening/") + "circularImport2.cellml'; and\n"
    + " - component 'circular2' specifies an import from '" + resourcePath("modelflattening/") + "circularImport2.cellml' to '" + resourcePath("modelflattening/") + "circularImport1.cellml'.";

const std::string flattenError =
    std::string("Cyclic dependencies were found when attempting to flatten a component in the model 'importExample2b'. The dependency loop is:\n")
    + " - component 'sideB' specifies an import from ':this:' to '" + resourcePath("modelflattening/") + "circularImport1.cellml';\n"
    + " - component 'shared' specifies an import from '" + resourcePath("modelflattening/") + "circularImport1.cellml' to '" + resourcePath("modelflattening/") + "circularImport2.cellml'; and\n"
    + " - component 'circular2' specifies an import from '" + resourcePath("modelflattening/") + "circularImport2.cellml' to '" + resourcePath("modelflattening/") + "circularImport1.cellml'.";
```

Reply:
has this been checked with clang format.

Reply:
Yes, using ClangFormat 12.